### DISCUSSION TITLE
Cursor input stopping for a brief second

### DISCUSSION THREAD

**Discussion by Legoshibrpro** [ppy/osu#15802] [Q&A] [answered]
recently ive switched to being a lazer main and noticed that quite frequently my cursor just stops completely for a second or a bit less out of the blue, such problem dont happen with the normal osu and also didnt happen a few months ago when i tested lazer for the first time with the same mouse. Pc, mouse specs, logs and a video showing the bug can be found here:

ps. after analyzing the footage again it looks like it also stops keyboard imput too.

Mouse - hyperX Pulsefire FPS Pro
OS- microsoft windows 10 home
Processor- Intel(R) Core(TM) i5-8265U CPU @ 1.60Hz, 1800 Mhz, 4 Core(s), 8 Logical Processor(s)
ram- 8gb

https://user-images.githubusercontent.com/79376264/143458879-b19713ea-07d2-4e74-a201-efa811b44cdf.mp4

i am also running on the latest osu!lazer version from the time of creating this post, which is 2021.1120.0-lazer 

i am also avaible to giving any more information that is not my credit card's :3

**Comment by bdach** [comment, marked as answer]
i cannot view the video attached.

either way please attach another one with the frame graph showing (press <kbd>Ctrl</kbd>+<kbd>F11</kbd> twice). i think it is very likely that this will be due to your mouse's polling rate being 1000 Hz. similar issues have been reported several times already.

**Reply by integerrr** [reply]
I re-encoded his video

https://user-images.githubusercontent.com/7279624/143506364-a38e53b8-c1de-4ec1-802b-42b408b293e8.mp4

**Reply by peppy** [reply]
Could you please re-make the video with frame  graphs shown as above?

**Reply by Legoshibrpro** [reply]
here, i re made it with the frame graph showing and this time with logs attatched, im sorry for forgetting to add them before

ps. the logs might not be helpful because i closed osu after the bug happened, but i can play a bit more and get the logs from when it happens since its quite frequent

https://user-images.githubusercontent.com/79376264/143581297-3cfb10e7-c2bd-4368-b887-fae4dfb033ea.mp4

[runtime.log](https://github.com/ppy/osu/files/7608728/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7608729/updater.log)
[database.log](https://github.com/ppy/osu/files/7608730/database.log)
[network.log](https://github.com/ppy/osu/files/7608731/network.log)
[performance.log](https://github.com/ppy/osu/files/7608732/performance.log)

**Reply by integerrr** [reply]
First of all, you only pressed ctrl-F11 once, please do another video with the full frame graph open by pressing ctrl-F11 again, you're looking for something like this:

<img width="458" alt="image" src="https://user-images.githubusercontent.com/7279624/143584787-d2dd1f94-78d5-41e8-b1cf-3dc931c01558.png">

Secondly, this video gives no information about the lag because you are only recording a REPLAY of the lag, which simply shows the cursor not moving but the game is not lagging, you need to re-record the video in gameplay and reproduce the lag with the frame graph open

**Reply by Legoshibrpro** [reply]
im sorry for the errors, and regarding making another video in real time, i havent been able to replicate the bug for a while, if it doesnt happen in like ~5 more days i will consider it fixed

**Reply by peppy** [reply]
It's good practice to not call such an issue a "bug" since it's likely something else on your PC causing it.

**Reply by Legoshibrpro** [reply]
sure, thank you three for the help, imma set this as fixed since it has not happened since
